.circleci/config.yml

```
@@ -31,6 +39,12 @@ workflows:
       - publish_to_npm:
           filters:
             tags:
-              only: /^v.*/
+              only: /^v[0-9]*\.[0-9]*\.[0-9]*$/
```

Currently it would match `v..`, `v1..`, and `v1.1.`, which is an invalid semver version. If we change the `*` to `+` then it would require atleast a single digit in each section, which is valid semver :-)
```suggestion
              only: /^v[0-9]+\.[0-9]+\.[0-9]+$/
```